users/winzou/authorized_keys

```diff
@@ -0,0 +1,2 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDzf0GPvALieM8BKdnjQvMl6yF1zlrxvX460k6cFsir7xpZ8uamkNv4dKcAuXvn+DSmb0oDQ4G/agpsKdm5/bpDz54I74O1vhkYxLf9oWrM2xChPwIGwJRBZ6/jdBfVA/1O8KsoiVwryySe8pljFN3fH6FvXeT60tMkkDJh4PM69nJgkoOFtmwlVQVuUKw5LQX6h/Ac3gMW0JYb3QDm+a3Aekj/GNls9poYL0L/DwSJWNUMHPFJDQQbvCV1ojZ4vdhyrv9AXS0PIM3l4Irwv4PlZocM2gPOgJ9qsbiF5mQx6ZcutLShG6679eB9NTsGIafbdCJi3+I+0I0uprErNbiHyV2p+cjAI17B9McOXg04f7hJvEZKl7YjURtm7LkFtAoAWt4oG3H+7k6tGmsuKUQJsSvoC5dXpCDK3+qMH4/JS0cb+y3lA2FEu/rC3r8sqZiAHg0X9likIbqzoUW4uQOpHb0kVH2xnaCdk64fbWL1qhJ7Wefn1/ptahg5jXsv5WE= emmet@boywhatdahe
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDzf0GPvALieM8BKdnjQvMl6yF1zlrxvX460k6cFsir7xpZ8uamkNv4dKcAuXvn+DSmb0oDQ4G/agpsKdm5/bpDz54I74O1vhkYxLf9oWrM2xChPwIGwJRBZ6/jdBfVA/1O8KsoiVwryySe8pljFN3fH6FvXeT60tMkkDJh4PM69nJgkoOFtmwlVQVuUKw5LQX6h/Ac3gMW0JYb3QDm+a3Aekj/GNls9poYL0L/DwSJWNUMHPFJDQQbvCV1ojZ4vdhyrv9AXS0PIM3l4Irwv4PlZocM2gPOgJ9qsbiF5mQx6ZcutLShG6679eB9NTsGIafbdCJi3+I+0I0uprErNbiHyV2p+cjAI17B9McOXg04f7hJvEZKl7YjURtm7LkFtAoAWt4oG3H+7k6tGmsuKUQJsSvoC5dXpCDK3+qMH4/JS0cb+y3lA2FEu/rC3r8sqZiAHg0X9likIbqzoUW4uQOpHb0kVH2xnaCdk64fbWL1qhJ7Wefn1/ptahg5jXsv5WE= emmet@boywhatdahe
```

Comment:
This appears to be the same key twice fyi. Did you mean to add two distinct keys?